docs/guides/scheduler_event.md

```diff
@@ -0,0 +1,107 @@
+# The scheduler event
+
+The scheduler event from Concord is the best way to get the RAW JSON that Discord sends to Concords, with it you can allow doing a lot of things, for example, you can make Concord ignore an event, or even better, just parse a JSON key manually to achieve the best performance from it.
```

Comment:
```suggestion
The Scheduler is one way to get the raw JSON from Discord's requests. This feature gives the developer fine-grain control over how an event behaves such that events can be ignored or spawn a worker thread. Additionally, the JSON key can be manually parsed to achieve the best performance.
```
Your details are there, but one way run-on sentences can be avoided is by noticing when comma splices (or using commas to start a new sentence) start to crop up.

Reply:
Thanks! But the scheduler does not retrieve the Discord's request JSON, only the Discord's websocket payload.